pkg/grouper/package_grouper.go

```diff
@@ -37,6 +44,10 @@ func GroupByPURL(packageSources []models.PackageSource) map[string]models.Packag
 	return uniquePackages
 }
 
+func isLocationExtractSuccessfully(pkgInfos models.PackageInfo) bool {
+	return pkgInfos.Line.Start > 0 && pkgInfos.Line.End > 0 && pkgInfos.Column.Start > 0 && pkgInfos.Column.End > 0
```

Comment:
The location of a library may be in the first line of a lockfile, and this check would discard such valid location